graspy/match/gmp.py

```diff
@@ -296,6 +296,7 @@ def f(x):  # computing the original optimization function
                 score = score_new
                 perm_inds = np.zeros(n, dtype=int)
                 perm_inds[permutation_A] = permutation_B[perm_inds_new]
+                best_n_iter = n_iter
```

Comment:
bit of a nit pick but I would prefer `n_iter_best` rather than `best_n_iter`

Reply:
Will do, I can change that. Sklearn used best_n_iter: 
<img width="434" alt="Screen Shot 2020-09-12 at 9 44 46 AM" src="https://user-images.githubusercontent.com/32661399/92997963-b0896c00-f4dc-11ea-936d-42eee7ffe93e.png">
Would you still like me to change it

Reply:
You can leave as is then.